spec/graphql/mutations/wallets/create_spec.rb

```diff
@@ -36,6 +39,12 @@
           grantedCredits: '0.00',
           expirationAt: expiration_at.iso8601,
           currency: 'EUR',
+          recurringTransactionRules: [
+            {
+              ruleType: 'interval',
+              interval: 'monthly',
+            },
+          ]
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.